parcel.go

```diff
@@ -49,12 +92,42 @@ func (s ParcelStore) SetAddress(number int, address string) error {
 	// реализуйте обновление адреса в таблице parcel
 	// менять адрес можно только если значение статуса registered
 
+	//	row := s.db.QueryRow("SELECT address, number FROM parcel WHERE number =:number",
+	//		sql.Named("address", address),
+	//		sql.Named("number", number))
+	//
+	//	if err := row.Err(); err != nil {
+	//		return err
+	//	}
+
+	_, err := s.db.Exec("UPDATE parcel SET address = :address WHERE number = :number and status = :status",
+		sql.Named("address", address),
+		sql.Named("number", number),
+		sql.Named("status", "registered"))
```

Comment:
Лучше использовать константу
```suggestion
		sql.Named("status", ParcelStatusRegistered))
```

Reply:
Исправил